### DISCUSSION TITLE
Show toolbar during when break/fail/pause overlay is visible

### DISCUSSION THREAD

**Discussion by rednir** [ppy/osu#14013] [Ideas]
People may not be aware of the options available to them during breaks or pauses, as the only way to access them is through a shortcut key.

In addition, mobile users obviously can't use shortcut keys and hence can't access these options at all without the toolbar.

Here's a video that shows the proposed change: https://streamable.com/p242dy
Relevant pull request: https://github.com/ppy/osu/pull/14026

**Comment by bdach** [comment]
While showing the toolbar during pause seems tolerable to me, I disagree about showing during breaks. It's quite distracting, especially with short pauses.
